Mathlib/Analysis/NormedSpace/Alternating/Uncurry/Fin.lean

```diff
@@ -0,0 +1,167 @@
+/-
+Copyright (c) 2025 Yury Kudryashov. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yury Kudryashov
+-/
+import Mathlib.Analysis.NormedSpace.Alternating.Curry
+import Mathlib.LinearAlgebra.Alternating.Uncurry.Fin
+
+/-!
+# Uncurrying continuous alternating maps
```

Comment:
The line breaks in this module docstring are a bit random, could you try to make them more homogeneous?